server/src/main/java/com/google/speech/tools/voxetta/servlets/PromptServlet.java

```diff
@@ -32,12 +33,17 @@
 @WebServlet("/prompt")
 public class PromptServlet extends HttpServlet {
 
-    private final DatastorePromptService promptService = new DatastorePromptService();
+    private final PromptService promptService = new DatastorePromptService();
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-        response.getWriter().write(promptService.getOnePrompt());
+        try {
+            response.getWriter().write(promptService.getOnePrompt());
```

Comment:
Same comment as before, `IOException` is built into the interface, so we don't specifically need to catch it. But if `promptService.getOnePrompt` throws any exception that should be caught and handled here